### PR TITLE
chore: adding allowed multiple locations

### PR DESCRIPTION
**Pull request by florianow**
Outcome of icf

### CODE REVIEW

File: kit/azure/organization-hierarchy/README.md [hunk 1]
```diff
@@ -71,23 +71,29 @@ After deploying this module, you should probably deploy the following kit module
 | Name | Type |
 |------|------|
 | [azurerm_management_group.connectivity](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/management_group) | resource |
+| [azurerm_management_group.corp](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/management_group) | resource |
 | [azurerm_management_group.identity](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/management_group) | resource |
 | [azurerm_management_group.landingzones](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/management_group) | resource |
 | [azurerm_management_group.management](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/management_group) | resource |
+| [azurerm_management_group.online](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/management_group) | resource |
```
Comment (JohannesRudolph) [inline kit/azure/organization-hierarchy/README.md:78]: d: iirc when we last discussed this you considered moving the online/corp groups out of the organization-hierarchy module and into the respective LZ modules?

File: kit/azure/organization-hierarchy/main.tf [hunk 2]
```diff
@@ -1,3 +1,8 @@
+locals {
+  default_location  = var.location[0]
+  allowed_locations = join(",", var.location)
```
Comment (JohannesRudolph) [inline kit/azure/organization-hierarchy/main.tf:3]: d: consider doing the join in the tmpl.json instead (as that's the only place that's concerned with taking a list into a json string). 

also: consider using the jsonencode function instead

File: kit/azure/organization-hierarchy/README.md [hunk 3]
```diff
@@ -71,23 +71,29 @@ After deploying this module, you should probably deploy the following kit module
...
 | <a name="input_landingzones"></a> [landingzones](#input\_landingzones) | n/a | `string` | `"lv-landingzones"` | no |
-| <a name="input_location"></a> [location](#input\_location) | The Azure location where this policy assignment should exist, required when an Identity is assigned. | `string` | `"germanywestcentral"` | no |
+| <a name="input_location"></a> [location](#input\_location) | The Azure location where this policy assignment should exist, required when an Identity is assigned. | `list(string)` | <pre>[<br>  "germanywestcentral"<br>]</pre> | no |
 | <a name="input_management"></a> [management](#input\_management) | n/a | `string` | `"lv-management"` | no |
+| <a name="input_online"></a> [online](#input\_online) | n/a | `string` | `"lv-online"` | no |
 | <a name="input_parentManagementGroup"></a> [parentManagementGroup](#input\_parentManagementGroup) | n/a | `string` | `"lv-foundation"` | no |
 | <a name="input_platform"></a> [platform](#input\_platform) | n/a | `string` | `"lv-platform"` | no |
+| <a name="input_stage_dev"></a> [stage\_dev](#input\_stage\_dev) | TODO: nor sure we want to keep stage dev in here | `string` | `"lv-stage-dev"` | no |
```
Comment (JohannesRudolph) [inline kit/azure/organization-hierarchy/README.md:96]: f: needs to go

File: kit/azure/organization-hierarchy/main.tf [hunk 4]
```diff
@@ -23,6 +29,24 @@ resource "azurerm_management_group" "landingzones" {
   parent_management_group_id = azurerm_management_group.parent.id
 }
 
+resource "azurerm_management_group" "stage_dev" {
```
Comment (JohannesRudolph) [inline kit/azure/organization-hierarchy/main.tf:32]: f: remove (fine to have it in ICF, but not in collie hub)

File: kit/azure/organization-hierarchy/main.tf [hunk 5]
```diff
@@ -9,10 +14,11 @@ module "policy_root" {
 
   policy_path         = "${path.module}/lib"
   management_group_id = azurerm_management_group.parent.id
-  location            = var.location
+  location            = local.default_location
 
   template_file_variables = {
-    default_location          = "${var.location}"
+    allowed_locations         = local.allowed_locations
```
Comment (JohannesRudolph) [inline kit/azure/organization-hierarchy/main.tf:20]: d: what do we need default_location vs. allowed_location for? this is not clear, can we name this more specifically?

**Review by JohannesRudolph — APPROVED** [review]

File: kit/azure/organization-hierarchy/README.md (hunk 1 above)
Comment (florianow) [reply kit/azure/organization-hierarchy/README.md:78]: copy paste fail sorry.

**Pull request merged**